teledata/serializers.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+from rest_framework.reverse import reverse
+from teledata.models import *
```

Comment:
Are all of these imports currently in use/necessary?

Reply:
`reverse` isn't, so I'll remove that. I just imported all of the models, should I just import the one for now? I'll be creating views for the others shortly

Reply:
👍 If you're going to be creating views for the rest of the teledata models soon, it makes sense to leave the global model import as-is.